openstates/oh/bills.py

```diff
@@ -13,43 +13,333 @@
+                bill_versions = {}
+                for doc in self.pages(base_url,first_page+doc_type):
+                    for v in doc["items"]:
+                        #bills is actually a list of versions
+                        #going to create a dictionary as follows:
+                        #key=bill_id
+                        #value = dict of all versions, where keys are versionids
+                        #and values are the bill data from the api.
+                        #then loop through that to avoid duplicate bills
+
+                        bill_id = v["number"]
+                        version_id = v["versionid"]
+                        if bill_id in bill_versions:
+                            if version_id in bill_versions[bill_id]:
+                                self.logger.warning("There are two versions of {bill_id} called the same thing. Bad news bears!".format(bill_id=bill_id))
+                            else:
+                                bill_versions[bill_id][version_id] = v
+                        else:
+                            bill_versions[bill_id] = {}
+                            bill_versions[bill_id][version_id] = v
+
+                    
+
+                for b in bill_versions:
+                    bill = None
+                    for bill_version in bill_versions[b].values():
+                        if not bill:
+                            bill_id = bill_version["number"]
+                            title = bill_version["shorttitle"] or bill_version["longtitle"]
+
+                            title = title.strip()
+
+                            chamber = chamber_dict[bill_version["chamber"]]
+
+                            subjects = []
+                            for subj in bill_version["subjectindexes"]:
+                                try:
+                                    subjects.append(subj["primary"])
+                                except KeyError:
+                                    pass
+
+
+                            bill = Bill(session,chamber,bill_id,title,subjects=subjects,type=doc_type)
+
+                            #this stuff is the same for all versions
+
+                            bill.add_source(first_page+doc_type+"/"+bill_id)
+
+                            sponsors = bill_version["sponsors"]
+                            for sponsor in sponsors:
+                                sponsor_name = self.get_sponsor_name(sponsor)
+                                bill.add_sponsor("primary",sponsor_name)
+
+                            cosponsors = bill_version["cosponsors"]
+                            for sponsor in cosponsors:
+                                sponsor_name = self.get_sponsor_name(sponsor)
+                                bill.add_sponsor("cosponsor",sponsor_name)
+
+                            #actions
+                            blacklist = ["/solarapi/v1/general_assembly_131/resolutions/lr_131_0035-1/actions"]
```

Comment:
I'm getting a 500 for `http://search-prod.lis.state.oh.us//solarapi/v1/general_assembly_131/resolutions/lr_131_0019/actions` as well. I'm surprised that the default `self.accept_response()` isn't handling this better, but maybe you could implement [something like in AL](https://github.com/sunlightlabs/openstates/blob/master/openstates/al/bills.py#L49)?